core/src/main/java/com/linecorp/armeria/common/metric/EventLoopMetrics.java

```diff
@@ -0,0 +1,133 @@
+/*
+ *  Copyright 2023 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.common.metric;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
+
+import com.linecorp.armeria.common.util.Ticker;
+import com.linecorp.armeria.internal.common.metric.MicrometerUtil;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.SingleThreadEventLoop;
+import io.netty.util.concurrent.EventExecutor;
+
+/**
+ * A {@link MeterBinder} to observe Netty {{@link EventLoopGroup}}s. The following stats are currently
+ * exported per registered {@link MeterIdPrefix}.
+ *
+ * <p>
+ *  - "event.loop.num.workers" (gauge) - the total number of Netty's event loops
+ *  - "event.loop.pending.tasks" (gauge) - the total number of IO tasks waiting to be run on event loops*
```

Comment:
How about using `<ul>` to render the Javadoc properly?
```suggestion
 * <ul>
 *   <li>"event.loop.num.workers" (gauge) - the total number of Netty's event loops</li>
 *   <li>"event.loop.pending.tasks" (gauge)
 *   - the total number of IO tasks waiting to be run on event loops</li>
 * </ul>
```